src/config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  apiKey: "AIzaSyCq8D1BGLEI15rNOSV_7e4aFoPWjvxr6Dk",
+  discoveryDocs: 
+    ["https://sheets.googleapis.com/$discovery/rest?version=v4"],
+  spreadsheetId: "1Il9WDIhaV3U2MHJOjiFcYSkVOaFpoIg1dge28HCPsrY"
+};
```

Comment:
not really sure about this, typically publishing API keys on public github repos is not recommended (usually it should be loaded from a .gitignored file or from an environmental variable or something like that). also, since we are potentially storing user's private information (if they supply it on the form), it would be a security risk.

Reply:
if you could add a commit:
* put the API key and ID into a text file
* add the text file to `.gitignore`
* before `export default` add some code that reads the key and ID from the text file, then adds the API key and ID to the exported object as variables
* squash commits to remove the previous version of the file from git history
and make sure that air partners can find the key and ID (add it to our shared Google Drive?) :D
@anushadatar do you have thoughts on this? my suggestions are based on my limited knowledge so maybe I'm way off base

Reply:
Good point. This was a sloppy mistake on my side. This file was for testing purposes and should not have made it to the pull request.

Reply:
@hsharriman  we don't even need the google API key or ID in the code base while we are using the current form submission method with google forms. I shouldn't have included it on my initial commit.

Reply:
Good catch on not wanting the ID here - I should have more carefully looked into what that string represented. 

The squash should mitigate the issues associated with having this private info in the git history if we also delete the branch - there's still an issue where a bad actor could manually find this private info in the pull request history, so I would argue it's worth doing a rebase, removing this file, and force-pushing to the branch. I understand if you don't think it's worth it and decide not to do it though. I'm also happy to help make this happen.

I think the text file based approach is totally fine here; it's a couple of steps but it's straightforward enough and something we can and should document somewhere private. Our google keep could be a great place for this information.